java/emitter.js

```diff
@@ -1,425 +1,484 @@
 'use strict'
+const types = require('./mapping/types')
 const {checkSettingsParser, java, getter, variable, call, returnSyntax, wrapSelector, parseEnv} = require('./parser')
 const {capitalizeFirstLetter} = require('./util')
-const imageMatchSettings = {
-  type: 'ImageMatchSettings',
-  schema: {
-    ignoreDisplacements: 'BooleanObject',
-    ignore: {type: 'Array', items: 'Region'},
-    floating: {type: 'Array', items: 'FloatingRegion'},
-    accessibility: {type: 'Array', items: 'AccessibilityRegion'},
-    accessibilitySettings: {
-      type: 'AccessibilitySettings',
-      schema: {level: 'AccessibilityLevel', version: 'AccessibilityGuidelinesVersion'},
+const ImageMatchSettings = {
+    type: 'ImageMatchSettings',
+    schema: {
+        ignoreDisplacements: 'BooleanObject',
+        ignore: {type: 'Array', items: 'Region'},
+        floating: {type: 'Array', items: 'FloatingRegion'},
+        accessibility: {type: 'Array', items: 'AccessibilityRegion'},
+        accessibilitySettings: {
+            type: 'AccessibilitySettings',
+            schema: {level: 'AccessibilityLevel', version: 'AccessibilityGuidelinesVersion'},
+        },
+        layout: {type: 'Array', items: 'Region'}
     },
-    layout: {type: 'Array', items: 'Region'}
-  },
+}
+const Location = {
+    type: 'Location',
+    schema: {
+        x: 'Number',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
                                                x: "Number",
```